wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/model/impl/WorkspaceRuntimeImpl.java

```diff
@@ -47,9 +47,11 @@ public WorkspaceRuntimeImpl(String activeEnv,
         if (devMachine != null) {
             this.devMachine = new MachineImpl(devMachine);
         }
-        this.machines = machines.stream()
-                                .map(MachineImpl::new)
-                                .collect(toList());
+        if (machines != null) {
```

Comment:
Is it possible that collection will be null? It is quite unusual for Che code base